tracer/test/snapshots/RabbitMQTests.6_x.SchemaV0.verified.txt

```diff
@@ -98,6 +98,7 @@
       env: integration_tests,
       language: dotnet,
       runtime-id: Guid_1,
+      server.address: rabbitmq,
```

Comment:
I think `server.address` needs to be refactored to `out.host` in all these files and the PR description, right?